src/entity/entityImpl.hpp

```diff
@@ -193,39 +193,31 @@ class LocalEntityImpl : public SuperClass, public protocol::ProtocolInterface::O
 		if (handler)
 			return std::bind(handler, object, std::forward<Ts>(params)...);
 		// No handler specified, return an empty handler
-		return [](LocalEntity::AemCommandStatus const /*error*/)
-		{
-		};
+		return [](LocalEntity::AemCommandStatus const /*error*/) {};
```

Comment:
I'd like to see that, good to know at least this is possible with the latest clang-format